src/app/services/account/update.js

```diff
@@ -0,0 +1,34 @@
+import Vue from 'vue';
+// import store from './../../store';
+
+// When the request succeeds
+const success = () => {
+  // store.dispatch('login', token);
```

Comment:
You still need to update the account in the store a bit like this:
```
updateAccount(store, account);
```

You need to import the newly created `updateAccount` action at the top:
```
import { updateAccount } from './../../store/actions';
```